charts/clusterplex/templates/worker.yaml

```diff
@@ -8,7 +8,7 @@ image:
   imagePullPolicy: '{{ .Values.global.plexImage.imagePullPolicy }}'
 
 controller:
-  type: statefulset
+  type: '{{ .Values.worker.config.type | default "statefulset" }}'
```

Comment:
Any chance you could add this new `worker.config.type` parameter and it's possible values to the chart's README.md (https://github.com/pabloromeo/clusterplex/blob/master/charts/clusterplex/README.md)?

Reply:
Done, however I used the `helm-docs` tool which generated a lot more changes than I was expecting. I could put my change in manually if you want.

Reply:
Ah, yeah, in fact you did the right thing. My instructions were incorrect.
Let me take a look tomorrow to see if the contents still looks correct and we'll merge it in.